### DISCUSSION TITLE
Asus Strix Z590-E Gaming Wifi (motherboard) & Asus TUF FX705GM (laptop) - Ventoy iso UEFI not working

### DISCUSSION THREAD

**Discussion by Maximebaker** [ventoy/Ventoy#1149] [Q&A] [answered]
Hi, I have a problem and I don't what to do anymore. 

Ventoy is booting correctly in UEFI mode. But after I select any ISO from the list, it boots the ISO in legacy mode.

I tried on an Asus t100ta and it was booting fine in UEFI. I also tried on an "old" Acer laptop, and it worked fine too. But not on my two Asus machine.

I tried with Easy2Boot with Ventoy, no success either.

Can someone help please?

**Comment by steve6375** [comment]
How do you know that the ISO is booting in legacy mode?

**Comment by steve6375** [comment, marked as answer]
If you boot to Windows 10 Setup and install Windows, if it booted in UEFI mode then GPT partitions will be made.

The BIOS logo only shows on a **BIOS** boot, but when you load the ISO from grub2 it is not booting from the BIOS but from grub2.

The resolution issue is a different problem. You can try pressing F7 before booting the ISO and see if that helps.

**Comment by steve6375** [comment]
See FAQ on ventoy.net site
